src/analytics/private/sqls/redshift/dashboard/clickstream_event_view_v3.sql

```diff
@@ -0,0 +1,145 @@
+  traffic_source_term,
+  traffic_source_campaign_id,
+  traffic_source_clid_platform,
+  traffic_source_clid,
+  traffic_source_channel_group,
+  traffic_source_category,
+  user_first_touch_time_msec,
+  app_package_id,
+  app_version,
+  app_title,
+  app_install_source,
+  platform,
+  project_id,
+  app_id,
+  screen_view_screen_name,
+  screen_view_screen_id,
+  screen_view_screen_unique_id,
+  screen_view_previous_screen_name,
+  screen_view_previous_screen_id,
+  screen_view_previous_screen_unique_id,
+  screen_view_previous_time_msec,
+  screen_view_engagement_time_msec,
+  case when screen_view_entrances then 'true' else 'false' end as screen_view_entrances,
+  page_view_page_referrer,
+  page_view_page_referrer_title,
+  page_view_previous_time_msec,
+  page_view_engagement_time_msec,
+  page_view_page_title,
+  page_view_page_url,
+  page_view_page_url_path,
+  page_view_hostname,
+  page_view_latest_referrer,
+  page_view_latest_referrer_host,
+  case when page_view_entrances then 'true' else 'false' end as page_view_entrances,
+  case when app_start_is_first_time then 'true' else 'false' end as app_start_is_first_time,
+  upgrade_previous_app_version,
+  upgrade_previous_os_version,
+  search_key,
+  search_term,
+  outbound_link_classes,
+  outbound_link_domain,
+  outbound_link_id,
+  outbound_link_url,
+  case when outbound_link then 'true' else 'false' end as outbound_link,
+  user_engagement_time_msec,
+  scroll_engagement_time_msec,
+  sdk_error_code,
+  sdk_error_message,
+  sdk_version,
+  sdk_name,
+  app_exception_message,
+  app_exception_stack,
+  custom_parameters_json_str,
+  custom_parameters,
+  e.session_duration,
+  s.session_number,
+  s.session_start_time_msec,
+  session_source,
+  session_medium,
+  session_campaign,
+  session_content,
+  session_term,
+  session_campaign_id,
+  session_clid_platform,
+  session_clid,
+  session_channel_group,
+  session_source_category,
+  u.first_touch_time_msec,
+  u.first_visit_date,
+  u.first_referrer,
+  u.first_traffic_category,
+  u.first_traffic_source,
+  u.first_traffic_medium,
+  u.first_traffic_campaign,
+  u.first_traffic_content,
+  u.first_traffic_term,
+  u.first_traffic_campaign_id,
+  u.first_traffic_clid_platform,
+  u.first_traffic_clid,
+  u.first_traffic_channel_group,
+  u.first_app_install_source,
+  u.user_properties_json_str,
+  u.user_properties,
+  CONVERT_TIMEZONE('{{timezone}}', e.event_timestamp)::DATE as event_date,
+  COALESCE(u.user_id, e.user_pseudo_id) as merged_user_id,
+  CASE WHEN event_name = '_first_open' THEN COALESCE(u.user_id, e.user_pseudo_id) ELSE NULL END as new_user_indicator,
+  CASE WHEN event_name IN ('_page_view', '_screen_view') THEN e.session_id ELSE NULL END as view_session_indicator,
+  CASE WHEN event_name IN ('_page_view', '_screen_view') THEN e.event_id ELSE NULL END as view_event_indicator,
+  DATE_TRUNC('second', CONVERT_TIMEZONE('{{timezone}}', e.event_timestamp)) AS event_timestamp_local
```

Comment:
Let's try not to introduce the timezone-awareness fields in this base view. It will cost a lot if changing the time zone.

Reply:
fixed